backend/src/apiserver/resource/resource_manager_test.go

```diff
@@ -96,7 +96,7 @@ func createPipelineVersion(pipelineId string, name string, description string, u
 	}
 	paramsJSON := "[{\"name\":\"param1\"}]"
 	spec := pipelineSpec
-	tmpl, err := template.New([]byte(pipelineSpec))
+	tmpl, err := template.New([]byte(pipelineSpec), true)
```

Comment:
Should the tests use `false` to keep the existing default behavior?

Reply:
Done.